internal/provider/data_file_test.go

```diff
@@ -0,0 +1,107 @@
+package provider_test
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/neuspaces/terraform-provider-system/internal/acctest"
+	"github.com/neuspaces/terraform-provider-system/internal/acctest/tfbuild"
+	"testing"
+)
+
+func TestAccFile_read_content(t *testing.T) {
```

Comment:
Rename to `TestAccDataFile_read_content` for naming consistency

Reply:
ok, done